extension/src/cli/git/reader.ts

```diff
@@ -68,6 +69,18 @@ export class GitReader extends GitCli {
     }
   }
 
+  public async getRemoteExperimentRefs(cwd: string): Promise<string> {
+    const options = getOptions({
+      args: [Command.LS_REMOTE, DEFAULT_REMOTE, 'refs/exps/*'],
```

Comment:
> There are no "remote watchers" to trigger table updates for things like other team members deleting your experiments from the remote or even a user running dvc exp remove -g origin <exp-name> themselves.

Might have already been discussed, but have we considered adding a refresh button that will run this command again? Could be a generic one that refreshes all table data entirely or adding an extra action to our "remote state" tooltips 🤔

Reply:
We have purposefully avoided a refresh button up to this point. We may have to introduce one for remote action but let's hold off until after we decide on a strategy for fetching the data.